.github/actions/spelling/allow.txt

```diff
@@ -533,6 +533,7 @@ testsettingname
 TEXTFORMAT
 TEXTINCLUDE
 there're
```

Comment:
Do we really need this one? It seems weird to me; I think we could change the one use of it to `there are`

Reply:
I agree, although apparently it is somewhat acceptable I have never seen it before.